templates/prescription/general_edit.html

```diff
@@ -256,10 +265,22 @@
 {$ENDING_JAVASCRIPT}
 </script>
 </form>
+<script type="text/javascript" src="{$WEBROOT}/public/assets/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="{$WEBROOT}/library/js/jquery-ui.min.js"></script>
 {literal}
 <!-- for the fancy jQuery stuff -->
 <script type="text/javascript">
 
+$(function() {
+    
+    //Drug autocomplete
+    $(".drug").autocomplete({
```

Comment:
No, it will not. These are two different auto-complete. The original autocomplete completes a secondary box that is dropped down below the actual form entry box. The other autocomplete you have to click the (click here to search) and it opens a div that the other autocomplete fills. Right now the drug search is broken. I need to fix it.